### PR TITLE
Allow methods returning null to be turned into expression bodied members

### PR DESCRIPTION
**Pull request by SSkovboiSS**
Hi
This pr fixes issue #271.

I have changed **ConvertExpressionBodyToStatementBodyCodeRefactoringProvider** so it will properly handle a case where expression was a void type and still was converting to a ReturnStatement.

I have also added a corresponding unit test that handle this case.

### CODE REVIEW

File: RefactoringEssentials/CSharp/CodeRefactorings/Synced/ConvertExpressionBodyToStatementBodyCodeRefactoringProvider.cs [hunk 1]
```diff
@@ -66,6 +66,14 @@ static void HandleMethodCase(CodeRefactoringContext context, SyntaxNode root, Sy
             ExpressionSyntax expr;
             if (!IsExpressionBody(method.Body, method.ExpressionBody, out expr))
                 return;
+            
+            var returnType = method.ReturnType as PredefinedTypeSyntax;
+            if (returnType == null)
```
Comment (siegfriedpammer) [inline RefactoringEssentials/CSharp/CodeRefactorings/Synced/ConvertExpressionBodyToStatementBodyCodeRefactoringProvider.cs]: What if the ReturnType is not predefined? Do we want to hide the refactoring in those cases?

Comment (SSkovboiSS) [inline RefactoringEssentials/CSharp/CodeRefactorings/Synced/ConvertExpressionBodyToStatementBodyCodeRefactoringProvider.cs]: Well yes, that would be not very good, just couldn't find out right way to check ReturnType property for void

**Comment by SSkovboiSS** [comment]
Improved a way of checking if method's return type is void

**Comment by SSkovboiSS** [comment]
Fixed issue #262 with adding checks if parent type is interface to appropriate refactoring providers

**Comment by siegfriedpammer** [comment]
Please create a separate pull request for the other bug fix.
AND
Please use `model.GetDeclaredSymbol(method)?.ReturnsVoid` instead of checking for the keyword.

**Pull request closed without merging**